app/models/user.rb

```diff
@@ -15,7 +15,7 @@ class User < ActiveRecord::Base
 
   attr_accessor :encrypted_password
 
-  before_create :encrypt_password
+  before_create :encrypt_password, :set_session_token
   before_update :encrypt_password
 
   has_many :tweets
```

Comment:
add this: has_many :liked_tweets, through: :user_like_tweets, class_name: 'Tweet'